api/ask_astro/app.py

```diff
@@ -18,6 +19,8 @@
 
 api = Sanic(name="ask_astro")
 
+CORS(api, resources={r"/api/*": {"origins": "*"}})
```

Comment:
In general it is not a good practice to allow `*` wildcard origin. Specify a domain name or something here.

But why are we adding this? The current service works just fine without it?

Reply:
We are not adding it @davidgxue it was just for the hackathon as we were using dev API of ask-astro

Reply:
Okay sounds good. As long as you don't merge this in then that's cool!